.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "files.autoSave": "afterDelay"
```

Comment:
What does this do? Maybe we shouldn't commit `.vscode/settings.json` because that will affect everyone's Visual Studio code settings. Can you please add this to `.gitignore`.